src/nrtk_explorer/app/ui/image_list.py

```diff
@@ -297,10 +331,10 @@ def __init__(self, on_scroll, on_hover, **kwargs):
                                         "Original: Detection Annotations",
                                         classes="text-center",
                                     )
-                                    ImageDetection(
+                                    ImageWithSpinner(
                                         identifier=("props.row.original",),
-                                        src=("props.row.original_src",),
-                                        annotations=("props.row.originalAnnotations",),
+                                        src="props.row.original_src",
```

Comment:
don't we want to have `:src="..."` or `v-bind:src="..."`? If so we should use the tuple notation

Reply:
never mind, you did it in the Python layer of ImageWithSpinner